crates/node-core/src/version.rs

```diff
@@ -1,7 +1,11 @@
 //! Version information for reth.
-
 use reth_db::models::client_version::ClientVersion;
 
+// The client code for Reth
+pub const CLIENT_CODE : &str = env!("RH");
+// The human readable name of the client 
+pub const NAME_CLIENT: &str = env!("reth");
```

Comment:
```suggestion
pub const CLIENT_CODE: &str = "RH";
// The human readable name of the client 
pub const NAME_CLIENT: &str = "reth";
```

this should not use `env!`